include/configs/mx6ullevk_siklu.h

```diff
@@ -18,15 +18,12 @@
 #include "mx6_common.h"
 #include <asm/mach-imx/gpio.h>
 
-#define is_mx6ul_9x9_evk()	CONFIG_IS_ENABLED(TARGET_MX6UL_9X9_EVK)
```

Comment:
This and other fixes to commits in the same series should be squashed into the commits they are fixing.